packages/driver/src/cypress/screenshot.ts

```diff
@@ -136,6 +136,7 @@ const validate = (props, cmd, log) => {
   validateAndSetBoolean(props, values, cmd, log, 'scale')
   validateAndSetBoolean(props, values, cmd, log, 'disableTimersAndAnimations')
   validateAndSetBoolean(props, values, cmd, log, 'screenshotOnRunFailure')
+  validateAndSetBoolean(props, values, cmd, log, 'overwrite')
```

Comment:
nit: could DRY this up:

```suggestion
  ;['scale', 'disableTimersAndAnimations', 'screenshotOnRunFailure', 'overwrite'].map(key =>
  	validateAndSetBoolean(props, values, cmd, log, key)
  )
```